### ISSUE TITLE
Fix "array" to "string" in Split Strings with Split

### ISSUE THREAD

**Issue by erictleung** [FreeCodeCamp/FreeCodeCamp#8510]
#### Challenge Name

https://www.freecodecamp.com/challenges/split-strings-with-split

#### Issue Description

The description has a typo.

> Here is an example of `split` being used to split **an array** at every `s` character:

But it should be:

> Here is an example of `split` being used to split **a string** at every `s` character:

#### How to Fix

[This line in the code](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json#L570) needs to be changed to:

```json
...
"Here is an example of <code>split</code> being used to split a string at every <code>s</code> character:",
...
```

#### Browser Information

* Browser Name, Version: Chrome
* Operating System: OSX
* Mobile, Desktop, or Tablet: Desktop

#### Screenshot

![image](https://cloud.githubusercontent.com/assets/2754821/15098459/6ec7df22-14f3-11e6-96b6-565909f80266.png)

**Comment by mdepak** [comment]
@erictleung Can I take this up?

**Comment by erictleung** [comment]
@mdepak go for it! Please read the [contributing guidelines](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md) before taking care of the issue and don't hesitate to visit the [Contributors Help](https://gitter.im/FreeCodeCamp/HelpContributors) if you have any questions about helping. Thanks for your interest!

**Comment by theflametrooper** [comment]
Closes #8510
